### ISSUE TITLE
sensor range error

### ISSUE THREAD

**Issue by ivoysey** [schmerl/LLStaging#54]
```[ WARN] [1488573223.078273231, 8.940000000]: The origin for the sensor at (-0.09, -0.04, 0.30) is out of map bounds. So, the costmap cannot raytrace for it.
```

**Comment by schmerl** [comment]
This only happens in CP2 right? I think this is because we start with the robot close to the wall, and so maybe the kinect sensor isn't right in this case. Or maybe it's a startup issue for the joints? In any case, if it goes away when we start driving, then I think this is ok.

**Comment by ivoysey** [comment]
Yeah it's only in CP2. I don't know if it goes away in motion. Maybe the
simulated Kinect floating above the tbot base is bumping the wall slightly?
The numbers printed out are not zero, although that's what the initial
config I was using specified, so that seems like either floating point
error or Gazebo being Helpful™.


---Ian

**Comment by schmerl** [comment]
I think this issue is ok, but maybe it will come out in testing.
